src/main/java/uk/gov/hmcts/reform/coh/schedule/notifiers/EventTransformerFactory.java

```diff
@@ -0,0 +1,37 @@
+package uk.gov.hmcts.reform.coh.schedule.notifiers;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.coh.events.EventTypes;
+
+import java.util.*;
+
+/**
+ * Utility class to manage all the interfaces in the app
+ */
+@Service
+public class EventTransformerFactory {
+
+    @Autowired
+    private Set<EventTransformer> eventTransformers;
+
+    /**
+     * Add all the Spring managed interfaces to an internal map
+     * @param transformers
+     */
```

Comment:
I originally wanted to map "keys" (event types) to transformers, but wasted 2 hours failing to get Spring to read EventTransformer.supports. As a workaround, I've done this crapping set this, then iterate the supports method to see which event type the transformer supports. I'll find a way to fix this more elegantly